lib/api/bulk.js

```diff
@@ -66,7 +66,17 @@ Job.prototype.info = function(callback) {
   // if cache is not available, check the latest
   if (!this._jobInfo) {
     this._jobInfo = this.check();
+  } else if (self.options.pkChunking) {
+    this._jobInfo.then((info) => {
```

Comment:
I think this should be returned? So that the caller chains off the result of `then` not the original `_jobInfo`.